packages/oid4vci-demo-frontend/src/router/AppRouter.tsx

```diff
@@ -4,22 +4,31 @@ import SSICredentialIssueRequestPage from '../pages/SSICredentialIssueRequestPag
 import SSIInformationSuccessPage from '../pages/SSIInformationSuccessPage';
 import SSICredentialVerifyRequestPage from '../pages/SSICredentialVerifyRequestPage';
 import SSICredentialIssuedSuccessPage from '../pages/SSICredentialIssuedSuccessPage';
-import SSILandingPage from '../pages/SSILandingPage';
-
 import SSIInformationRequestPage from '../pages/SSIInformationRequestPage';
 import SSIDownloadPage from "../pages/SSIDownloadPage";
+import {getCurrentEcosystemPageOrComponentConfig, SSICredentialsLandingPageConfig} from "../ecosystem-config";
+import SSICredentialsLandingPage from "../pages/SSICredentialsLandingPage";
 
 const AppRouter: React.FC = () => {
+    const config = getCurrentEcosystemPageOrComponentConfig('SSICredentialsLandingPage') as SSICredentialsLandingPageConfig
     return (
         <HashRouter>
             <Routes>
-                <Route path="/" element={<SSILandingPage/>}/>
+                {/*<Route path="/" element={<SSILandingPage/>}/>*/}
+                <Route path="/" element={<SSICredentialsLandingPage/>}/>
                 <Route path={"/information/request"} element={<SSIInformationRequestPage/>}/>
                 <Route path={"/information/success"} element={<SSIInformationSuccessPage/>}/>
                 <Route path={"/credentials/verify/request"} element={<SSICredentialVerifyRequestPage/>}/>
                 <Route path={"/credentials/issue/request"} element={<SSICredentialIssueRequestPage/>}/>
                 <Route path={"/credentials/issue/success"} element={<SSICredentialIssuedSuccessPage/>}/>
                 <Route path={"/download"} element={<SSIDownloadPage/>}/>
+                {config.credentials.map(value => (
```

Comment:
What is this? The minicard overview is a single page listing multiple credentials with static links to other environments. This looks like dynamic routes based on credential names for a single environment

Reply:
i have no clue what this is

Reply:
seems to be working with, so removed